src/WebUI/Pods/PodOverview.tsx

```diff
@@ -190,7 +191,7 @@ export class PodOverview extends BaseComponent<IPodOverviewProps> {
                     {value}
                 </Link>
             </Tooltip>
-            : defaultColumnRenderer(value, undefined, valueTooltipText);
+            : defaultColumnRenderer(value, undefined, imageDetailsUnavailableTooltipText);
```

Comment:
we should show "image name | image details not available" text only when promise is done, before promise is done we should not show this.

the algorithm we want is:
- before promise for image details are done:
1) show tooltip only when element is not fit in visible area, otherwise do not show any tooltip.

- when promise is done,
1) if we have image details, we will show the link. tooltip is shown only when the element is not fit.
2) if image details not available then we show text like "image name | not available (something like this)"

Reply:
accordingly please change the algorithm

Reply:
Done